releases/release-1.24/release-notes/maps/pr-106725-map.yaml

```diff
@@ -0,0 +1,4 @@
+pr: 106725
+releasenote:
+  text: Updated the default API priority-and-fairness config to avoid endpoint/configmaps
```

Comment:
The PR says 'Update default API priority-and-fairness config to avoid endpoint/configmaps operations from controller-manager to all match leader-election priority level.' So, changing it to this message..